string action_state
string robot_db
int16 tag_id
